### DISCUSSION TITLE
Is it still running?

### DISCUSSION THREAD

**Discussion by FokkerSF** [Prince25/StockAlertBot#48] [Q&A] [answered]
When I initially start StockAlertBot, I get notification that the item is not in stock and that it "will keep retrying in background every 31 seconds" But i never get future messages after that.  Is it still running?

**Comment by Prince25** [comment, marked as answer]
Yeah, it's always running and checking at whatever interval you set it. I should honesty make it possible to toggle messages. If you still have doubts, put in an URL for an in-stock item and you'll see the alert appear.
